package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "watch-detector",
-  "version": "1.0.0",
+  "version": "2.0.0",
```

Comment:
I don't believe you need to bump this version number yourself. This repo uses `release-it`, which will increment the version number on publish.